packages/agent-toolkit/src/lib/clerk-client.ts

```diff
@@ -0,0 +1,22 @@
+import { createClerkClient } from '@clerk/backend';
+import { apiUrlFromPublishableKey } from '@clerk/shared/apiUrlFromPublishableKey';
+
+const API_VERSION = process.env.CLERK_API_VERSION || 'v1';
```

Comment:
You can use `import { getEnvVariable } from '@clerk/shared/getEnvVariable'` now that Rob added in https://github.com/clerk/javascript/pull/4985

Reply:
Done!